static/js/ReaderPanel.jsx

```diff
@@ -41,7 +41,7 @@ import {
   ToggleSet, InterfaceText, EnglishText, HebrewText, SignUpModal,
 } from './Misc';
 import {ContentText} from "./ContentText";
-import SheetsHomePage from "./Sheets/SheetsHomePage";
```

Comment:
should be lowercase directory

Reply:
This changeset should have been in the dedicated pr for renaming the dir as well (see pr 1924 comments)